runtime/onert/backend/cpu/ops/TransposeLayer.h

```diff
@@ -44,6 +44,8 @@ class TransposeLayer : public ::onert::exec::IFunction
 
   void run();
 
+  void runSync() { run(); }
```

Comment:
Please check if this is in the scope of this PR. If not, please make another PR for this. (it's not a scope of shapeinf)

Reply:
I probably got this in the process of resolving a merge conflict. I will revert the changes after the test results. Thanks for spotting.